.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I'd say that our position should be not to check in editor-specific code, and a wiki page would be most welcome.